jobs/func_job.go

```diff
@@ -0,0 +1,74 @@
+package jobs
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/flanksource/commons/logger"
+	"github.com/robfig/cron/v3"
+)
+
+type funcJob struct {
+	name    string                      // name is just an additional context for the job.
+	timeout time.Duration               // optional timeout for the job
+	fn      func(context.Context) error // the actual job
+	runNow  bool                        // whether to run the job now
```

Comment:
Add comments to top of fields instead of inline

Reply:
This is a fairly standard practice.  For short descriptions, I prefer this because I can still get the fields formatted in line.